source/PlanetLabel.cpp

```diff
@@ -39,6 +40,41 @@ namespace {
 	const double LINE_GAP = 1.7;
 	const double GAP = 6.;
 	const double MIN_DISTANCE = 30.;
+
+	// Check if the given label for the given stellar object and direction overlaps with any other stellar object in the system.
+	bool Overlaps(const System &system, const StellarObject &object, double zoom, double width, int direction)
+	{
+		bool overlaps = false;
+
+		Point start = object.Position() * zoom +
+			(object.Radius() + INNER_SPACE + LINE_GAP + LINE_LENGTH) * zoom * Angle(LINE_ANGLE[direction]).Unit();
```

Comment:
Can't this be simplified now?
```suggestion
		Point start = zoom * (object.Position() +
			(object.Radius() + INNER_SPACE + LINE_GAP + LINE_LENGTH) * Angle(LINE_ANGLE[direction]).Unit());
```